app/controllers/idv/socure/document_capture_controller.rb

```diff
@@ -66,15 +66,20 @@ def update
         # Not used in standard flow, here for data consistency with hybrid flow.
         document_capture_session.confirm_ocr
 
+        # If the stored_result is nil, the job fetching the results has not completed.
+        if stored_result.nil?
+          analytics.idv_doc_auth_document_capture_polling_wait_visited(**analytics_arguments)
+          render 'shared/wait'
+          return
+        end
```

Comment:
thanks for adding logic to handle this async result fetching. using this approach, does the user stay in an infinite loop if we never fetch the result? 🤔